force-app/main/default/pages/PrimaryContact.page

```diff
@@ -0,0 +1,65 @@
+<apex:page standardController="Account" extensions="searchAndPick">
+    <script>
+    
+    function javascriptmethod(Id){
```

Comment:
please use more descriptive name for function and for parameters, use camel case